apps/web/configuration/tailwind.config.ts

```diff
@@ -7,6 +7,11 @@ import { fontFamilyText } from './fontFamily.config';
 export default {
   presets: [tailwindConfig],
   content: ['./**/*.vue', '../../node_modules/@storefront-ui/vue/**/*.{js,mjs}'],
+  safelist: [
+    {
+      pattern: /text-\w+-\d+/,
```

Comment:
```suggestion
      pattern: /text-\w+(-(1|2|3|4|5|6|7|8|9)0{1,2})?/,
```
**issue**: This pattern doesn't satisfy the `text-black` default. Same goes for `text-white`, which is probably the most common alternative.

Reply:
Yes I forgot about these classes, but I would choose something like this instead:
/text-\w+(-\d{2,3})?/

Will work with all colors available right now:
text-black
text-white
text-orange-50
text-orange-600

Reply:
Sure, that's probably also fine.